versioneer.py

```diff
@@ -10,7 +10,7 @@
 * https://github.com/warner/python-versioneer
 * Brian Warner
 * License: Public Domain
-* Compatible With: python2.6, 2.7, 3.3, 3.4, 3.5, and pypy
+* Compatible With: python3.5+, conda, and pypy
```

Comment:
We should not be changing this, since `versioneer.py` isn't really part of this package.

Reply:
I forgot to undo this when I un-did the other ones, but it's done now